### ISSUE TITLE
Persian/Arabic languages

### ISSUE THREAD

**Issue by alr86** [Anuken/Mindustry-Suggestions#5366]
### Describe the content or mechanics you are proposing.

I have completed the translation of the game into Persian. However, during testing, I discovered that the game does not support Persian (or Arabic) characters. Although the sample below is not finalized for use in the game, I am sharing it for testing purposes.

Please implement support for Persian and Arabic languages
Thanks!

[bundle_fa.properties](https://shz.al/~bundle_fa-properties)

> Mindustry doesn't support Arabic and other RTL languages (e.g. Hebrew), because libGDX >doesn't.
>
> https://discord.com/channels/391020510269669376/395414970793459714/789997183494782986
> https://github.com/libgdx/libgdx/issues/787
>
> _Originally posted by @joshuaptfan in https://github.com/Anuken/Mindustry-Suggestions/issues/1981#issuecomment-779870415_

I have seen some fixes for it like [this](https://github.com/ultra-deep/libgdx-rtl-support) or [this](https://github.com/iibrahimbakr/libGDX-RTL-Language) in GitHub

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by camelStyleUser** [comment]
> Could anuke please suffer for a small part of the userbase?

No.You should suffer and make it instead.You can PR support for RTL yourself into Arc and Mindustry.
